packages/cli/generation/ir-migrations/src/migrations/v20-to-v19/migrateFromV20ToV19.ts

```diff
@@ -25,6 +25,7 @@ export const V20_TO_V19_MIGRATION: IrMigration<
         [GeneratorName.STOPLIGHT]: AlwaysRunMigration,
         [GeneratorName.POSTMAN]: AlwaysRunMigration,
         [GeneratorName.PYTHON_SDK]: "0.3.6-rc1-2-g6050f7f8",
+        [GeneratorName.GO_MODEL]: AlwaysRunMigration,
```

Comment:
i'm guessing the go model generator will depend on the latest version of the IR, should we put `0.0.0` from here? Basically from the onset the go generator will not support ir version 15 and this will make sure of that

Reply:
Discussed offline - we actually want to use `undefined` in all of the migration files to prevent any migration from occurring - `fern-go-model` will only accept `v20+` to start.